portfolio/src/main/webapp/script.js

```diff
@@ -153,9 +153,11 @@ function createMarketMap() {
       {position: {lat: market.lat, lng: market.lng}, map: map}
     );
 
-    const marketContent = "Market's name: " + market.name + ". Website: " + market.website;
+    const marketContent = "<b>Market's name:</b> " + market.name + ".";
+    const websiteContent = market.website.startsWith("http") ? 
+      (" <b>Website:</b> " + "<a href='" + market.website + "'>" + market.website + "</a>") : "";
```

Comment:
Add a new line before ":". The formatting should look like this:

```
const x = <condition> 
  ? <very long value.....>
  : <other value>
```

Reply:
Great, thanks. Done.